src/engine/source/api/src/test/handlers.cpp

```diff
@@ -458,6 +467,113 @@ api::Handler sessionsDelete(shared_ptr<Router> router, shared_ptr<Catalog> catal
     };
 }
 
+api::Handler runPost(std::shared_ptr<::router::Router> router)
+{
+    return [router](api::wpRequest wRequest) -> api::wpResponse
+    {
+        using RequestType = eTest::RunPost_Request;
+        using ResponseType = eTest::RunPost_Response;
+        const auto res = ::api::adapter::fromWazuhRequest<RequestType, ResponseType>(wRequest);
+
+        // If the request is not valid, return the error
+        if (std::holds_alternative<api::wpResponse>(res))
+        {
+            return std::move(std::get<api::wpResponse>(res));
+        }
+
+        eTest::DebugMode debugMode;
+        uint32_t protocolQueue;
+        std::string protocolLocation;
```

Comment:
We could declare these when assigning them and even make them const if possible.